api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -424,6 +424,12 @@ message HttpConnectionManager {
   // Note that Envoy does not perform
   // `case normalization <https://tools.ietf.org/html/rfc3986#section-6.2.2.1>`
   google.protobuf.BoolValue normalize_path = 30;
+
+  // If present and set to true, the connection manager will validate the values of each HTTP
+  // header, returning a 400 response if any of the headers contain an invalid
+  // value. See `Header Fields `<https://tools.ietf.org/html/rfc7230#section-3.2>`_
+  // for details.
+  google.protobuf.BoolValue validate_headers = 33;
```

Comment:
Normally we use the POD boolean type when the default is false

Reply:
Sure, sounds good to me.  I think I had been looking at some [other fields on the conn manager](https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/filter/network/http_connection_manager/v2/http_connection_manager.proto#envoy-api-msg-config-filter-network-http-connection-manager-v2-httpconnectionmanager) like `add_user_agent`, which uses the proto wrapper type, but defaults to false.